action/protocol/staking/handlers.go

```diff
@@ -62,7 +64,15 @@ func (p *Protocol) handleCreateStake(ctx context.Context, act *action.CreateStak
 		return nil, errors.Wrapf(err, "failed to store account %s", actionCtx.Caller.String())
 	}
 
-	receipt, err := p.settleAction(ctx, sm, gasFee)
+	log := &action.Log{
+		Address:     p.addr.String(),
+		Topics:      nil,
+		Data:        byteutil.Uint64ToBytes(bucketIdx),
```

Comment:
In rewarding protocol, data is struct with type, address, and amount. To be consistent with rewarding protocol, we may need to do the same thing in the staking protocol. However, we won't be able to do indexing or bloomfilter with the complex data structure. thoughts?